_partials/jbds-is.adoc

```diff
@@ -2,11 +2,11 @@
 
 1.  Install the Java SE Development Kit (JDK), we recommend OpenJDK or
 Oracle JDK.
-2.  Download link:#{site.products['devstudio'].downloads['8.1.0.GA'].assets[site.products['devstudio'].default_download_artifact_type].artifacts[0].url}[JBoss Developer Studio 8.1.0.GA #{site.products['devstudio'].downloads['8.1.0.GA'].assets[site.products['devstudio'].default_download_artifact_type].artifacts[0].name}]
+2.  Download link:https://developers.redhat.com/download-manager/content/origin/files/sha256/1e/1efef4d2b109efd9e8136d02067a8d62e00621e5b2b1530811bf52b5aa2fca6f/jboss-devstudio-9.0.0.GA-CVE-2015-7501-installer-eap.jar?tp=fuse[JBoss Developer Studio 9.0.0.GA]
 3.  Run JBoss Developer Studio installer +
 For Mac / Windows Development Hosts:
 a.  Go to the folder you have downloaded JBoss Developer Studio:
-...  Right Click on *jboss-devstudio-8.1.0.GA-installer-standalone.jar*
```

Comment:
This should be changed the 8.1.0 eap bundle jar, not the 9.0 one.

https://developers.redhat.com/download-manager/file/jboss-devstudio-8.1.0.GA-jar_universal.jar